Remove the "#error please port this file to linux 2.6.18" line that
was left over from the port to 2.6.18.
===================================================================
diff -uNpr linux-2.6.19.2.orig/kernel/ctswic.h linux-2.6.19.2/kernel/ctswic.h
--- linux-2.6.19.2.orig/kernel/ctswic.h	2007-04-25 17:28:14.000000000 -0600
+++ linux-2.6.19.2/kernel/ctswic.h	2007-04-25 17:37:47.000000000 -0600
@@ -1,4 +1,3 @@
-#error "please port this file to linux 2.6.18"
 /*
 -------------------------------------------------------------------------------|
 |
